docs/source/user/cfunc.rst

```diff
@@ -194,11 +194,6 @@ Due to ABI limitations, structures should be passed as pointers
 using ``types.CPointer(my_struct)`` as the argument type.  Inside the ``cfunc``
 body, the ``my_struct*`` can be accessed with ``carray``.
 
-Full example
-------------
-
-See full example in ``examples/notebooks/Accessing C Struct Data.ipynb``.
```

Comment:
Should this link just be updated to https://github.com/numba/numba-examples/blob/master/legacy/notebooks/Accessing%20C%20Struct%20Data.ipynb ?

Reply:
I think it is best to include the documentation directly, at least in the long run. Yet a temporary solution could be just to update the link.